_sass/_header.scss

```diff
@@ -1,17 +1,7 @@
 header.site-header {
+  width: auto;
+  display: grid;
+  grid-template-columns: auto;
   background: #ffffff;
-  color: rgb(40, 40, 40);
-  .content-wrap {
-    display: flex;
-    font-family: 'Raleway';
-    font-size: x-large;
-    justify-content: space-around;
-    padding: 10px;
-   
-    a {
-      color: rgb(63, 63, 63);
-    }
-  }
+  color: #ffffff;
 }
-
```

Comment:
I know we haven't covered this, but wrapping is generally not a best practice since it can be more confusing to figure out when you come back to it, or if another dev looks at it later.
The way to refactor this is to unwrap these things.
So you'd have one class for 
.header.site-header { stuff }
And then another for header.site-header .content-wrap {stuff}
And yet another for header.site-header .content-wrap a {stuff}
If this doesn't make sense we can pair!

Reply:
We can find time to go over it. My brain needs to "unwrap" this because it is mush at 4pm.